portfolio/src/main/java/com/google/sps/controllers/Authenticator.java

```diff
@@ -33,10 +33,12 @@ public LoginInfo getLoginInfo(){
         LoginInfo loginInfo = new LoginInfo();
         loginInfo.isLoggedIn = userService.isUserLoggedIn();
           if(loginInfo.isLoggedIn){
-              loginInfo.url = userService.createLogoutURL(RETURN_URL);
-              loginInfo.nick = datastoreInterface.getUserNick(userService.getCurrentUser().getUserId());
+                com.google.appengine.api.users.User apiUser = userService.getCurrentUser();
```

Comment:
1. Please import namespace to not use FQ name (just simply `User`).
2. Indentation should be fixed. Also, one line below, there is space missing before =